network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,79 @@ These release notes track the development of the Network Observability Operator
+
+[id="network-observability-zones"]
+==== Availability zones
+You can configure the `FlowCollector` to collect information about the cluster availability zones. This allows you to enrich network flow data with the link:https://kubernetes.io/docs/reference/labels-annotations-taints/#topologykubernetesiozone[`topology.kubernetes.io/zone`] label value applied to the nodes. For more information, see xref:../network_observability/observing-network-traffic.adoc#network-observability-zonesnw-observe-network-traffic[Working with availability zones].
+
+[id="network-observability-enhanced-configuration-and-ui-1.5"]
+==== Notable enhancements 
+
+[discrete]
+[id="web-console-enhancements-1.5_{context}"]
+===== Web console enhancements:
+
+* New panels added to the *Overview* view for DNS & RTT: Min, Max, P90, P99.
+* New panel display options to focus on one panel while keeping others viewable but smaller focus, switch graph type, and show *Top* and *Overall*.
+* Collection latency warning in the *Custom time range* pop-up window.
+* Enhanced visibility for *Manage panels* and *Manage columns* pop-up windows contents.
+* The Differentiated Services Code Point (DSCP) field for egress QoS is now available for filtering QoS DSCP in the web console *Network Traffic* page.
+
+[discrete]
+[id="configuration-enhancements-1.5_{context}"]
+===== Configuration enhancements:
+
+The 1.5 release of the Network Observability Operator adds improvements and new capabilities to the {product-title} web console plugin and the Operator configuration.
+
+* The `LokiStack` mode in the `spec.loki.mode` specification simplifies installation by automatically setting URLs, TLS, cluster roles and a cluster role binding, as well as the `authToken` value. The `Manual` mode allows more control over configuration of these settings.
+* The API version changes from `flows.netobserv.io/v1beta1` to `flows.netobserv.io/v1beta2`.
+
+[id="network-observability-operator-1.5.0-bug-fixes"]
+=== Bug fixes
+
+* Previously, when the `FlowCollector` resource had the `spec.console.register` value set to false, meaning that the console plugin is not automatically registered, registering the console plugin manually via the Console interface would not work, as the operator would override and erase plugin registration.
+With this fix, setting the `spec.console.register` value to false does not impact the console plugin registration or registration removal, meaning that the plugin can be safely registered by other means. (link:https://issues.redhat.com/browse/NETOBSERV-1134[*NETOBSERV-1134*])
+* Previously, using the default metrics settings, the *NetObserv/Health* dashboard was showing an empty graph named "Flows Overhead". You could still get this metric by removing "namespaces-flows" and "namespaces" from the list of ignored tags. Now, using the default metrics settings, this metric is visible. (link:https://issues.redhat.com/browse/NETOBSERV-1351[*NETOBSERV-1351*])
```

Comment:
```suggestion
* Previously, the *NetObserv/Health* dashboard displayed an empty graph named *Flows Overhead* if you used the default metrics setting. The *Flows Overhead* metric was still available if you removed `namespaces-flows` and `namespaces` from the list of ignored tags. Now, using the default metrics settings, this metric is visible. (link:https://issues.redhat.com/browse/NETOBSERV-1351[*NETOBSERV-1351*])
```

I am not sure about how to label the graph/metric. It might be worth looking at the ISG for guidance.

Reply:
All the graphs are treated as GUI items in the web console, so it gets bold treatment.